app/js/controllers/service.js

```diff
@@ -3,12 +3,27 @@
 import { Controller } from 'components/fxos-mvc/dist/mvc';
 
 import Service from 'js/views/service';
+import CameraService from 'js/views/camera-service';
 
 export default class ServiceController extends Controller {
   main(id) {
```

Comment:
Ah, okay, will read about these lifecycle methods. So let me try to do something like this.